### PR TITLE
Main mobile

### PR DESCRIPTION
**Pull request by hydaline-code**
I hope you're doing well. Please kindly give feedback review  ♻ to my portfolio setup project where I implemented the header and the headline sections. Thanks

### CODE REVIEW

**Review by Ofarouq310 — COMMENTED** [review]
Hi @hydaline-code , ✋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 🥈 

giphy

## 🔴 Invalid Code Review Request  

- Your Pull request title should be short, and concise, and addresses what you are implementing, in your case, it could be, `Add header and headline sections` or something similar, simpler. Please refer to the next picture to help you how to edit the title.
image

-  As for the PR description, you should simplify it even more, just summarizing what you added in a bullet list or number list, take a look at this example, to edit the description, just click on three dots on the right side, then click on edit.
image



- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <h1 class="myname">Charlie 
+        Giorgania
+        </h1> 
+        <p  class="headline-p">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br> <br>
+  
+          If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      <button class="headline-btn">
+        Start collaboration
+      </button>
```
Comment (bobb-Rob) [inline index.html:38]: You have done very well implementing this project without any serious issues with your code. Well done. 

- [ ] I noticed a little issue with the `Start Collaboration`, it has a gray color border that should be removed. This border is a default style of buttons. Kindly add `border: none;` to the `.headline-btn` class. 

Your button | Figma Button
---- | ----
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <div class="headline-img">
+        <img src="./images/image_geometry_2.png" alt="intro image">
+      </div>
```
Comment (bobb-Rob) [inline index.html]: - [ ] This section's background image should be applied to the page using the CSS `background-image` property. Usually, if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.


```html
<div class="headline-img">
        <img src="./images/image_geometry_2.png" alt="intro image">
      </div>
```

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @hydaline-code, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @hydaline-code , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Design 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**